generate-summary.py

```diff
@@ -41,14 +41,23 @@
 		i += 1
 
 # just copy values to jure.xls
+result = []
 for t in total:
 	cnt = sum(t[1])
 	if cnt == 0:
 		continue
 	if rel:
 		out=""
+		out2 = []
 		for c in t[1]:
 			out+= str(float(c*100) / float(cnt))[:4] + " " 
+			out2.append(round(((c*100)/cnt),1))
 		print(out)
+		result.append(out2)
 	else:
 		print(t[0], t[1], cnt)
+
+# Save results to csv file 
+with open(os.getcwd()+"\\assets\\data\\cat-shares-new.csv", "w", newline='') as f:
```

Comment:
Windows understands forward slashes, so in the future just use `/` as path separators in python code — it'll be cleaner.